File: app/src/main/java/io/intrepid/skotlinton/rest/RestApi.java
package io.intrepid.skotlinton.rest;

import io.intrepid.skotlinton.models.IpModel;
import io.reactivex.Observable;
import retrofit2.http.GET;

public interface RestApi {
    @GET("/?format=json")
    Observable<IpModel> getMyIp();
}

File: app/src/main/kotlin/io/intrepid/multiskeleton/rest/RestApi.kt
package io.intrepid.multiskeleton.rest

import io.intrepid.multiskeleton.models.IpModel
import io.reactivex.Single
import retrofit2.http.GET

interface RestApi {
    @GET("/?format=json")
    fun getMyIp(): Single<IpModel>
}
